docs/_slides/iTzalin78.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from [username]!
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
```

Comment:
❓ Please review for suggestions @unaizorrilla

Reply:
Ummm, I prefer this new image for you

```suggestion
![dinotocat](https://octodex.github.com/images/minertocat.png)
```